spec/controllers/email_actions_controller_spec.rb

```diff
@@ -19,6 +19,24 @@
     end
   end
 
+  describe "follow_discussion" do
+    before do
+      @user = FactoryGirl.create(:user)
+      @group = FactoryGirl.create(:group)
+      @group.add_member!(@user)
+
+      @discussion = FactoryGirl.build(:discussion, group: @group)
+      DiscussionService.create(discussion: @discussion, actor: @user)
+      DiscussionReader.for(discussion: @discussion, user: @user).unfollow!
+    end
+
+    it 'follows the discussion' do
+      DiscussionReader.for(discussion: @discussion, user: @user).following?.should be false
```

Comment:
```
expect(DiscussionReader.for(discussion: @discussion, user: @user).following?).to be_false
```

Hm, this `.should` thing's popped up frequently enough recently that I'm thinking we might need to put some prominent wording in the 'devving on Loomio' wiki around it.